ext/vulnsrc/alpine/alpine_test.go

```diff
@@ -15,28 +15,24 @@
 package alpine
 
 import (
-	"os"
 	"path/filepath"
 	"runtime"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestYAMLParsing(t *testing.T) {
 	_, filename, _, _ := runtime.Caller(0)
 	path := filepath.Join(filepath.Dir(filename))
+	secdb, err := parseSecDBFile(path + "/testdata/v34_main.yaml")
```

Comment:
We should use `filepath.Join()` again here instead of `+`.
 While it's obvious now, we you refactor and change code, it becomes less obvious over time.